apps/experiments/versioning.py

```diff
@@ -78,8 +89,16 @@ def get_field(self, field_name: str) -> VersionField:
 
     @property
     def fields_grouped(self):
-        # TODO: Return fields with group info for display purposes
-        pass
+        groups = defaultdict(dict)
+        for field in self.fields:
+            group_name = field.group_name
+            group_info = groups.get(group_name, FieldGroup(name=group_name))
```

Comment:
you could use `setdefault` so that you don't have to insert it into the dict at the end:

```suggestion
            group_info = groups.setdefault(group_name, FieldGroup(name=group_name))
```